x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -425,7 +425,7 @@ HTTP method to use when making requests. `GET` or `POST` are the options. Defaul
 [float]
 ==== `request.encode_as`
 
-ContentType used for encoding the request body. If set it will force the encoding in the specified format regardless of the `Content-Type` header value, otherwise it will honor it if possible or fallback to `application/json`. By default the requests are sent with `Content-Type: application/json`. Supported values: `application/json` and `application/x-www-form-urlencoded`. `application/x-www-form-urlencoded` will url encode the `url.params` and set them as the body. It is not set by default.
+ContentType used for encoding the request body. If set it will force the encoding in the specified format regardless of the `Content-Type` header value, otherwise, it will honor it if possible or fallback to `application/json`. By default, the requests are sent with `Content-Type: application/json`. Supported values: `application/json` and `application/x-www-form-urlencoded`. `application/x-www-form-urlencoded` will url encode the `url.params` and set them as the body. It is not set by default.
```

Comment:
I don't think the first comma is wanted, the second one can be added, but is strictly not needed.

Reply:
I made the changes through manual inspection and with the assistance of Grammarly. Yes, I agree with your point. Removing the commas is also fine as per the following:

From Grammarly
> An introductory word (Unfortunately), phrase (Before the party), or clause (If you can join us) sets the stage for the sentence. **Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not.**

Should I remove the commas?

Reply:
Yes please.